.github/workflows/build.yml

```diff
@@ -11,110 +15,66 @@ on:
       - master
       - dev
 
-
 jobs:
   build:
-    runs-on: ${{ matrix.os }} 
+    runs-on: ${{ matrix.os }}
     strategy:
+      fail-fast: false
       matrix:
         os: [ ubuntu-latest, macos-latest, windows-latest ]
-    
+
     steps:
-      - name: checkout
-        uses: actions/checkout@v2
-      - name: get node
-        uses: actions/setup-node@v1
-        with:
-          node-version: 11.x
-      - name: linux setup
+      - name: Windows setup
+        if: ${{ matrix.os == 'windows-latest' }}
+        run: |
+          curl -LO https://downloads.arduino.cc/arduino-1.8.19-windows.zip
+          7z x arduino-1.8.19-windows.zip -o"$Env:TEMP\arduino-ide"
+          echo "$Env:TEMP\arduino-ide\arduino-1.8.19" | Out-File -FilePath $env:GITHUB_PATH -Append
```

Comment:
Would it be worthwhile to do a hash check on the download as well (like we're doing in the ADO pipeline)?

Reply:
Added for Linux and Windows. I'm leaving Mac alone for now because it installs through Homebrew. We could likely switch it to something closer to the Linux install, but I'll leave that for a future PR.